core/src/main/java/com/graphhopper/routing/util/DataFlagEncoder.java

```diff
@@ -217,6 +224,15 @@ int getAccessValue(ReaderWay way) {
             }
         }
 
+        if(accessValue == 0){
+            GHPoint estmCentre = way.getTag("estimated_center", null);
+            if (estmCentre != null) {
+                SpatialRule rule = spatialRuleLookup.lookupRule(estmCentre);
+                accessValue = getEdgeValueForAccess(rule.isAccessible(way, ""));
+            }
+
+        }
```

Comment:
Yes, the idea was to first check the ReaderWay if there is an access restriction value on it. If not check the country rules to eventually add a restriction.